### ISSUE TITLE
Weird behavior on Firefox with small window while dragging.

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/graphing-quadratics#164]
For https://github.com/phetsims/qa/issues/692, reported by @brooklynlash via Slack DM. She said:

> I found that there is some weird text thing going on when on Firefox and the screen is small, for example the size or a bit bigger than the iframe. I think this is just an issue with the browser itself but I wanted to make sure it wasn't an issue. It is easiest to see in this gif in the borders of the top right text box.

![graphingquadfirefox](https://user-images.githubusercontent.com/3046552/130358421-6322fa89-48e6-4d75-b50e-995c0e5b72b5.gif)

> Oh and this is when moving the draggable pieces only.
>
> Also, I am only seeing this amplified this much on the first two screens only, with the graphs that aren't draggable

**Comment by pixelzoom** [comment]
@brooklynlash Just to make sure that I understand...  While you're dragging the point tools, all of the graphical elements shift around slightly. And some line widths appear to get thicker, like the borders on panels.  Is that correct?  Anything else that I missed from watching the GIF?

And could you also please put into in this issue about your test platform? (machine, OS, browser, etc.)

**Comment by pixelzoom** [comment]
@brooklynlash Could you also please try some other sims, using phettest?  I'd like to establish that this is a general problem, and not specific to Graphing Quadratics.

**Comment by pixelzoom** [comment]
I could not reproduce this on macOS 11.5 (latest) with Firefox 89.0.2.   I tried with iframe and no iframe, using the RC links: 
https://phet-dev.colorado.edu/html/graphing-quadratics/1.2.0-rc.1/phet/graphing-quadratics_all_iframe_phet.html
https://phet-dev.colorado.edu/html/graphing-quadratics/1.2.0-rc.1/phet/graphing-quadratics_all_phet.html.

I then update to Firefox 91.0.1 (latest) and still could not reproduce.

**Comment by pixelzoom** [comment]
@amanda-phet can you please comment on whether this is blocking for Graphing Quadratics 1.2 release?  My feeling is that it is not blocking, and is highly likely to be a general "graphics artifact" issue that affects all sims.

**Comment by amanda-phet** [comment]
I would like to confirm that it's a general issue and not sim-specific, but that is my gut feeling as well. And when that is the case, I prefer not to let it block the specific sim, and handle it in a way that fixes it for all sims.

**Comment by brooklynlash** [comment]
@pixelzoom I was on Win10 Firefox, but have seen similar behavior on Chromebook as well. 

I can try other sims on phettest if you think it still needs confirmation that it is not sim-specific. 
Let me know what else you need.

**Comment by pixelzoom** [comment]
@brooklynlash Yes, please test a few other sims.  As @amanda-phet mentioned in https://github.com/phetsims/graphing-quadratics/issues/164#issuecomment-904066545, we'd like to establish whether this is a general issue, or specific to this sim. (I suspect it's general.)

**Comment by KatieWoe** [comment]
It does seem to happen with other sims. I was checking published. It didn't look exactly the same on my machine, even on graphing-quadratics, and this is likely because it is a bit dependent on the resolution. But shifting of panel lines and even sometimes text on Firefox does not seem to be sim dependent.
